### PR TITLE
Update spi to v0.6.0

### PR DESCRIPTION
**Pull request by skabashnyuk**
Include :
 - Removed file-retriver Demo UI. It was not properly configure on staging and never used.
 - https://github.com/redhat-appstudio/service-provider-integration-operator/pull/128
 - https://github.com/redhat-appstudio/service-provider-integration-operator/pull/138
 - https://github.com/redhat-appstudio/service-provider-integration-oauth/pull/99
 - https://github.com/redhat-appstudio/service-provider-integration-operator/pull/146
 - https://github.com/redhat-appstudio/service-provider-integration-operator/pull/133
 - https://github.com/redhat-appstudio/service-provider-integration-operator/pull/147
 - https://github.com/redhat-appstudio/service-provider-integration-operator/pull/151
 - https://github.com/redhat-appstudio/service-provider-integration-operator/pull/148

### CODE REVIEW

File: components/spi/kustomization.yaml [hunk 1]
```diff
@@ -14,16 +12,12 @@ kind: Kustomization
...
     newName: quay.io/redhat-appstudio/service-provider-integration-operator
-    # 0.5.4
-    newTag: sha-7abc780
+    # 0.6.0
+    newTag: sha-be71407
   - name: quay.io/redhat-appstudio/service-provider-integration-oauth
     newName: quay.io/redhat-appstudio/service-provider-integration-oauth
-    # 0.5.6
-    newTag: sha-bc6979a
-  - name: quay.io/redhat-appstudio/service-provider-integration-scm-file-retriever-server
```
Comment (sparkoo) [inline components/spi/kustomization.yaml:23]: no more file retriever ?

**Review by sparkoo — APPROVED** [review]

File: components/spi/kustomization.yaml (hunk 1 above)
Comment (skabashnyuk) [reply components/spi/kustomization.yaml:23]: yeh. it was never working. I've decided to remove it to not to confuse people.

**Pull request merged**